app/views/optOut/OptOutChooseTaxYear.scala.html

```diff
@@ -98,6 +102,10 @@
         @messages("optout.chooseOptOutTaxYear.desc")
     }
 
+    @if(submissionCounts.isQuarterlyUpdatesMade) {
```

Comment:
Small comment, can we get a unit test in the OptOutChooseTaxYearViewSpec for the warning inset. 
The test setup scenario at first glance looks like it should have a quarterly update so it should be present but it may need a new test scenario. not completely sure tho.

Reply:
yup I can look into that
the change was tested to ensure it was dynamic but it would probably be a good idea to have an actual written test for it - especially given the scoverage readings